### DISCUSSION TITLE
Sound Pressure Level Capture Trigger modification

### DISCUSSION THREAD

**Discussion by IgnacioFerrando** [gopro/labs#547] [General]
Hi
I'm new using Labs and a bit confused as I'm not a programmer... 
basically I have some Hero 11 mini... and I need to use a loud sound as trigger to start and stop recording... so I saw the Sound Pressure Level Capture trigger but is not what I need because I want the sound to trigger the start and stop so if the camera is not recording it starts recording when the sound happens and don't stop until another sound happens again...
can anyone help?
is there a tutorial on how to use the visual scripting?
thanks

**Comment by dnewman-gpsw** [comment]
No tutorials, as it is so new, you will just have to experiment.   I think what you describe is possible, but it more a logic flow problem. 

Here is my crude attempt (no testing)
![image](https://github.com/gopro/labs/assets/10856084/d3d636ca-2165-4451-b45e-4c754a308cc8)

You can try, and load this PNG in the visual scripting tool to modify.
![Clap Record Trigger-9_19_2023, 09_02_00](https://github.com/gopro/labs/assets/10856084/c9b94d8c-2d7c-4f58-9fd9-87d06da55265)

**Reply by IgnacioFerrando** [reply]
it works!! thank you VERY much
